src/main/kotlin/commands/AnnounceCommand.kt

```diff
@@ -0,0 +1,30 @@
+package commands
+
+import Command
+import ConfigManager.readConfigSafe
+import Send.error
+import UserConfig
+import greedyString
+import arg
+import doesLater
+
+object AnnounceCommand : Command("announce") {
+    init {
+        greedyString("content"){
+            doesLater{context ->
+                val content: String? = context arg "content"
+                if (content.isNullOrEmpty()) {
+                    message.error("You need to put announcement here.")
+                    return@doesLater
+                }
+                Main.client?.servers?.find(message.server?.id ?: run {
+                    message.error("Run this command inside a server!")
```

Comment:
this literally can't happen

Reply:
It can happen.

Reply:
No, it cannot, because server cannot be null. The bot doesn't listen to messages in dms

Reply:
Server can be null if ran in a DM

Reply:
It cannot be run in a dm.